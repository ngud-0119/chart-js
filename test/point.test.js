import React from 'react'
import renderer from 'react-test-renderer'
import BareMinimum2d from 'bare-minimum-2d'

const container = {
    color: '#0000FF',
    opacity: 0.2,
    xRange: 300,
    yRange: 500
}

const data = [{
    x: [0],
    y: [-20],
    color: "#FFFFFF",
    opacity: 1.0,
    size: 10,
    type: 'points',
    id: 'sample'
}]

it('renders correctly', () => {
    const tree = renderer
        .create(<BareMinimum2d {...{ data, container }} />)
        .toJSON()
    expect(tree).toMatchSnapshot()
})
