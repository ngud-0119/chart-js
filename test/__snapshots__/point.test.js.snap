// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  baseProfile="full"
  height="100%"
  preserveAspectRatio="xMidYMid slice"
  version="1.1"
  viewBox="0 0 300 500"
  width="100%"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    fill="#0000FF"
    height="100%"
    opacity={0.2}
    width="100%"
  />
  <circle
    cx={150}
    cy={270}
    fill="#FFFFFF"
    id="sample-0"
    opacity={1}
    r={10}
  />
</svg>
`;
